contracts/cryptoblades.sol

```diff
@@ -73,8 +77,10 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
     int128 public fightRewardBaseline;
     int128 public fightRewardGasOffset;
 
+    int128 public burnWeaponFee;
     int128 public mintWeaponFee;
     int128 public reforgeWeaponFee;
+    int128 public reforgeWeaponWithDustFee;
```

Comment:
Because of how upgradable proxy contracts work, we cannot add new state variable in the middle of other ones like this. They must be added *after all other state variables in the contract*.